lib/post/widgets/post_view.dart

```diff
@@ -143,10 +143,22 @@ class _PostSubviewState extends State<PostSubview> with SingleTickerProviderStat
                   if (thunderState.postBodyViewType == PostBodyViewType.condensed && !thunderState.showThumbnailPreviewOnRight && postViewMedia.media.first.mediaType != MediaType.text)
                     _getMediaPreview(thunderState, hideNsfwPreviews, markPostReadOnMediaView, isUserLoggedIn),
                   Expanded(
-                    child: ScalableText(
-                      HtmlUnescape().convert(post.name),
-                      fontScale: thunderState.titleFontSizeScale,
-                      style: theme.textTheme.titleMedium,
+                    child: Column(
+                      mainAxisSize: MainAxisSize.min,
+                      mainAxisAlignment: MainAxisAlignment.start,
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: [
+                        ScalableText(
+                          HtmlUnescape().convert(post.name),
+                          fontScale: thunderState.titleFontSizeScale,
+                          style: theme.textTheme.titleMedium,
+                        ),
+                        if (postViewMedia.media.first.mediaType == MediaType.link && thunderState.postBodyViewType == PostBodyViewType.condensed)
+                          Text(
+                            Uri.tryParse(post.url ?? '')?.host ?? '',
```

Comment:
Do you want to replace `www.` here too?

https://github.com/thunder-app/thunder/blob/cc8c44abf1fbd004ebdcf34f4689af82a2a7f185/lib/community/widgets/post_card_metadata.dart#L402

Reply:
Done!